internal/pkg/deploy/cloudformation/stack/testdata/workloads/rdws-manifest.yml

```
@@ -56,6 +56,8 @@ environments:
       LOG_LEVEL: debug        # Log level for the "test" environment.
       DB_NAME:
         from_cfn: test-UserDB
+    secrets:                      # Pass secrets from AWS Systems Manager (SSM) Parameter Store and Secrets Manager.
+      my-ssm-secret: "arn:aws:ssm:us-east-1:123456789123:parameter/jan11ssm"
```

nit: Can we also do `from_cfn` and `secretsmanager` to see how it renders?
```suggestion
    secrets:                      # Pass secrets from AWS Systems Manager (SSM) Parameter Store and Secrets Manager.
      my-ssm-secret: "arn:aws:ssm:us-east-1:123456789123:parameter/jan11ssm"
      MY_FROM_CFN_SECRET:
        from_cfn: my-other-stack-exported-secret
      MY_SECRETS_MANAGER_SECRET:
        secretsmanager: mysecretsmanagername
```

---

Addressed in next commit. Renders fine!